docs-src/faq.md

```diff
@@ -1,6 +1,74 @@
 # Frequently Asked Questions (FAQ)
 
-*   Q: Will there be a default controller implementation (in this repo)?<br>
+*   **Q: How can I get involved with Service API?<br>**
+    A: The [community](/community) page keeps track of how to get
+    involved with the project.
+
+*   **Q: Will Service API replace the Ingress API?<br>**
+    A: No. The Ingress API is GA since Kubernetes 1.19. There are no
+    plans to deprecate this API and we expect most Ingress controllers
+    to support it indefinitely.
+
+*   **Q: What are the differences between Ingress and Service API?<br>**
+    A: Ingress primarily targets exposing HTTP applications with a
+    simple, declarative syntax. Service API exposes a more general API
+    for proxying that can be used for more protocols than just HTTP,
+    and models the proxying infrastucture to provide better deployment
```

Comment:
I question whether we should use the word "proxy" as it may be interpreted by the reader as an implementation detail or requirement.